### ISSUE TITLE
"relu_layer.cu:29] Cuda kernel failed. Error: invalid configuration argument" on ilsvrc12

### ISSUE THREAD

**Issue by caffecuda** [BVLC/caffe#282]
Hi,

We got the master branch on March 28, while the mnist and cifar10 demos worked fine on Tesla M2090, the ilsvrc12 training script gave the following error:

...
I0401 15:13:18.902433  3143 net.cpp:173] Collecting Learning Rate and Weight Decay.
I0401 15:13:18.902487  3143 net.cpp:166] Network initialization done.
I0401 15:13:18.902521  3143 net.cpp:167] Memory required for Data 210114408
I0401 15:13:18.902660  3143 solver.cpp:36] Solver scaffolding done.
I0401 15:13:18.902760  3143 solver.cpp:47] Solving CaffeNet
F0401 15:13:19.040458  3143 relu_layer.cu:29] Cuda kernel failed. Error: invalid configuration argument
**\* Check failure stack trace: ***
    @     0x2b06c5dcbb7d  google::LogMessage::Fail()
    @     0x2b06c5dcdc7f  google::LogMessage::SendToLog()
    @     0x2b06c5dcb76c  google::LogMessage::Flush()
    @     0x2b06c5dce51d  google::LogMessageFatal::~LogMessageFatal()
    @           0x48188c  caffe::ReLULayer<>::Forward_gpu()
    @           0x431ada  caffe::Net<>::ForwardPrefilled()
    @           0x423cb8  caffe::Solver<>::Solve()
    @           0x40e645  main
    @     0x2b06c806176d  (unknown)
    @           0x40fced  (unknown)
Aborted (core dumped)
Done.

Is this related to the issue described in https://github.com/BVLC/caffe/issues/12?
